### PR TITLE
missing error solved

### LINKED ISSUE #815: Missing Links
### Description

image

> **At the bottom right of the homepage, there are logos for Facebook, Instagram, LinkedIn, and Twitter. However, they currently redirect to the same page because the appropriate links are not provided.** 

> Can u please assign me to this issue so i can start working on ti?

### Current behaviour

> currently rediecting it to the home page

### Expected behaviour

> redirected to the appropriate links

### Reproduction steps

```bash
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
```

### Screenshots

DESCRIPTION

### Logs

_No response_

### Browsers

_No response_

### OS

Windows

### Are you contributing for a program ?

Other

### Code of Conduct

- [X] I have read and agree to the Contributing Guidelines of this project.

### PR DESCRIPTION
**Pull request by Tanmay-Joshi73**
## Related Issue

image

Closes: #815

> Hey i have solved the error and also  add the the Discord logo with its appropraite link

## Checklist:

- [✔ ] My code adheres to the established style guidelines of this project.
- [✔ ] I have conducted a self-review of my code.
- [ ] I have included comments in areas that may be difficult to understand.
- [ ✔] I have made corresponding updates to the project documentation.
- [ ] My changes have not introduced any new warnings.

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Please adjust to your needs (see https://www.gitpod.io/docs/introduction/learn-gitpod/gitpod-yaml)
+# and commit this file to your remote git repository to share the goodness with others.
+
+# Learn more from ready-to-use templates: https://www.gitpod.io/docs/introduction/getting-started/quickstart
+
+tasks:
+  - init: npm install && npm run build
+    command: npm run start
+
+
```
Comment (anmode) [inline .gitpod.yml:11]: why this?

File: public/sitemap-0.xml [hunk 2]
```diff
@@ -1,24 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
-<url><loc>https://grabtern.com</loc><lastmod>2023-08-08T16:06:19.390Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
-<url><loc>https://grabtern.com/auth/forgotpass</loc><lastmod>2023-08-08T16:06:19.390Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
-<url><loc>https://grabtern.com/auth/login</loc><lastmod>2023-08-08T16:06:19.390Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
```
Comment (anmode) [inline public/sitemap-0.xml:5]: this file needs not to be changed...could you rever this and put this into gitignore!

**Review by anmode — COMMENTED** [review]
Some changes rest looks good!

File: public/sitemap-0.xml (hunk 2 above)
Comment (Tanmay-Joshi73) [reply public/sitemap-0.xml:5]: ok

**Pull request closed without merging**